credmark/cmf/credmark_dev.py

```diff
@@ -203,8 +203,9 @@ def load_models(args, load_dev_models=False):
 
 
 def show_version(_args):
-    ver = version('credmark-model-framework')
-    print(f'credmark-model-framework version {ver}')
```

Comment:
This was here so we could also show versions on server etc.

Reply:
Reverted to main.